lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -451,6 +451,9 @@ def _configure_node(
             self.vm_disks_dir, f"{node_context.vm_name}-os.qcow2"
         )
 
+        if node_runbook.disk_size_gb:
+            node_context.os_disk_size_in_gb = node_runbook.disk_size_gb
```

Comment:
It doesn't need to make a copy in context, unless it needs to change the value. You can get runbook anytime from the node.

Reply:
This is removed now as corresponding code is platform specific as Chirs suggested to push it to _create_node_os_disk(). I used runbook as suggested to get the resize param.